autoload/go/complete.vim

```diff
@@ -153,9 +153,9 @@ function! go#complete#GetInfo()
     return go#complete#GetInfoFromOffset(offset)
 endfunction
 
-function! go#complete#Info()
+function! go#complete#Info(auto)
     let result = go#complete#GetInfo()
-    if !empty(result)
+    if !empty(result) && (!a:auto && result ==# "PANIC PANIC PANIC")
```

Comment:
Can you please add a comment here what `auto` means and what `Panic Panic Panic` means? (I know what it is, but we have to much logic here and if someone else reads the code there is zero information)